package.json

```diff
@@ -155,6 +155,7 @@
         "@testing-library/cypress": "7.0.0",
         "@types/redux-devtools": "3.0.47",
         "@types/redux-devtools-extension": "2.13.2",
+        "clipboardy": "^2.3.0",
```

Comment:
why do we need this?

Reply:
this is used to get the clipboard from node to check if the copy worked. The cypress test is currently deactivated as firefox does not copy in cypress currently, but chrome does work. I committed the test commented out for the time when firefox can copy things in cypress. It's used in the plugin.js file in cypress

Reply:
So you created a test, that isn't activated because it doesn't work?
IMHO it would be better to create an issue (and add a todo to the test?), that says "create the test as soon as cypress+firefox is ready for clipboard"?
Because right now we have an unused dependency.

Reply:
I created a test with stuff I found in issues about the problem, so I'm very reluctant to remove that working test. If you don't like the dependency we could remove it and add it to the comment in the test.

Reply:
Well but the test doesn't work because the code is deactivated, right?

Reply:
But if that test would work it wouldn't be deactivated , right? 😅

Reply:
As stated above one of the tests does not work with firefox, because privacy and cypress did not (yet) build a workaround for that.

Reply:
Other than that the test is working flawless.

Reply:
I did what I suggested